src/main/java/prts/Prts.java

```diff
@@ -0,0 +1,65 @@
+package prts;
+
+import prts.command.Command;
+import prts.command.CommandParser;
+
+/**
+ * A chatbot and to-do list that can receive text input from the user, parse it, and execute
+ * predefined commands.
+ */
+public class Prts {
+
+    private final Storage storage;
+    private TaskList tasks;
+    private final Ui ui;
+
+    private String fileName = "prts.PRTS.txt";
+    private String[] directories = {"data"};

```

Comment:
Why is it being stored as an array?

Reply:
It's possible that the user might want to create the file in a nested directory somewhere, so I wanted to accept multiple directory names as inputs to enable such a functionality in the future. Though I do agree that as of now, this is indeed a bit redundant as this field is effectively final.

E.g. If `directories = {"deeply", "nested", "data", "folder"};`, I'd like to be able to create the save file in the path `user.dir/deeply/nested/data/folder/`.